.changeset/spicy-carpets-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-t3-app": patch
+---
+
+Added op:image, set up headSeo for correct usage. Set config site to 'beta.create.t3.gg'
```

Comment:
Not needed for docs changes.

Changesets only for the CLI

Reply:
Ok, removed the file.